resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/kubernetes/constants.scala

```diff
@@ -63,6 +63,7 @@ package object constants {
   private[spark] val ENV_EXECUTOR_MEMORY = "SPARK_EXECUTOR_MEMORY"
   private[spark] val ENV_APPLICATION_ID = "SPARK_APPLICATION_ID"
   private[spark] val ENV_EXECUTOR_ID = "SPARK_EXECUTOR_ID"
+  private[spark] val ENV_DRIVER_MEMORY = "SPARK_DRIVER_MEMORY"
```

Comment:
It's used in the `spark-class` script which launches the REST server itself: https://github.com/apache-spark-on-k8s/spark/blob/k8s-support-alternate-incremental/launcher/src/main/java/org/apache/spark/launcher/SparkClassCommandBuilder.java#L78